tests/Unit/NumericalAlgorithms/RootFinding/Test_TOMS748.cpp

```diff
@@ -9,45 +9,46 @@
 
 #include "DataStructures/DataVector.hpp"
 #include "ErrorHandling/Error.hpp"
-#include "NumericalAlgorithms/RootFinding/RootFinder.hpp"
+#include "NumericalAlgorithms/RootFinding/TOMS748.hpp"
+#include "Utilities/ConstantExpressions.hpp"
 
 namespace {
-double f_free(double x) { return 2.0 - x * x; }
+double f_free(double x) { return 2. - square(x); }
```

Comment:
It looks like the majority of this file uses `2.0`, could you make the whole file consistent? :)